package.json

```diff
@@ -24,13 +26,15 @@
     "morgan": "^1.9.1",
     "mysql2": "^2.0.0",
     "pug": "^2.0.4",
-    "sequelize": "^5.21.2"
+    "sequelize": "^5.21.2",
+    "dotenv": "^8.2.0"
```

Comment:
This should be a dev dependency.

Reply:
>     * `config.json` was renamed to `config.js` but it looks like it is never used

config.js is only used by the sequelize cli for migrations

Reply:
👌

Reply:
If this file must exist for the CLI, then we might as well use it for the app too. This will ensure the same env vars are always used. What do you think?